tutorials/ui/bbcode_in_richtextlabel.rst

```diff
@@ -492,8 +501,17 @@ Paragraph options
   | `Default` | Inherit                                    |
   +-----------+--------------------------------------------+
 
-  Locale override.
+  Locale override. Some font files may contain script-specific substitutes, in which case they will be used.
+
+- **tab_stops**
+
+  +-----------+------------------------------------------------+
+  | `Values`  | List of floating-point numbers, e.g. ``10,30`` |
```

Comment:
Maybe it's worth showing a floating point number here, so people aren't confused by the comma (it is used as a decimal separator in some languages/locales).